tools/packaging/static-build/agent/Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN apt-get update && \
 		musl-tools \
 		openssl \
 		perl \
-		protobuf-compiler && \
+		protobuf-compiler \
+		clang && \
```

Comment:
Why do we need clang?

Reply:
To build the agent with the guest pull feature, clang is required in the environment. Otherwise, the following error will occur: https://github.com/kata-containers/kata-containers/issues/9004. Since the agent can be built independently, clang needs to be installed only when building the agent.